### PR TITLE
fix(ext): log class for class constructor

### LINKED ISSUE #17197: Logging a class constructor logs a function rather than a class
When logging a class constructor, Deno prints `Function` but I expect `class`.

Create the following file `test.js`:

```js
class Foo {}
class Bar extends Foo {}
console.log(Foo, Bar);
```

Run it with Deno:

```sh
$ deno run test.js 
[Function: Foo] [Function: Bar]
```

This is unexpected. IMO Node does it better:

```sh
$ node test.js 
[class Foo] [class Bar extends Foo]
```

Is this intentional?

### PR DESCRIPTION
**Pull request by MarybethGasman**
fix #17197
add check for class constructor

### CODE REVIEW

**Comment by bartlomieju** [comment]
Nice fix @MarybethGasman! There are some tests that are failing due to this change (rightly so!), could you please update the output of relevant tests to make CI happy?

**Comment by MarybethGasman** [comment]
I think our CI is happy again!
cheers!
BTW, @tannal is me too. (seems like I have a broken git config.😀)

File: ext/console/02_console.js [hunk 1]
```diff
@@ -347,6 +347,14 @@
       // use generic 'Function' instead.
       cstrName = "Function";
     }
+    let stringValue = "";
+    if (typeof value.toString === "function") {
+      stringValue = value.toString();
+    }
+    // Might be Class
+    if (stringValue.startsWith("class")) {
```
Comment (petamoriken) [inline ext/console/02_console.js:355]: same here

```suggestion
    if (StringPrototypeStartsWith(stringValue, "class")) {
```

File: ext/console/02_console.js [hunk 2]
```diff
@@ -347,6 +347,14 @@
       // use generic 'Function' instead.
       cstrName = "Function";
     }
+    let stringValue = "";
+    if (typeof value.toString === "function") {
+      stringValue = value.toString();
+    }
```
Comment (petamoriken) [inline ext/console/02_console.js:353]: Please use primordials

```suggestion
    const stringValue = FunctionPrototypeToString(value);
```

**Review by petamoriken — CHANGES_REQUESTED** [review]

File: ext/console/02_console.js (hunk 2 above)
Comment (MarybethGasman) [reply ext/console/02_console.js:353]: Sounds good
Thanks for reminding me

**Review by bartlomieju — APPROVED** [review]
LGTM, great fix @MarybethGasman, thank you!

**Pull request merged**